common/persistence/persistence-tests/executionManagerTest.go

```diff
@@ -2670,7 +2671,7 @@ func (s *ExecutionManagerSuite) TestResetMutableStateCurrentIsSelf() {
 	s.Equal(int64(2333), ti.Version)
 	s.Equal("t1", ti.TimerID)
 	s.Equal(int64(1), ti.StartedID)
-	s.Equal(expiryTime.Unix(), ti.ExpiryTime.Unix())
+	s.True(timeComparator(expiryTime, ti.ExpiryTime, time.Millisecond))
```

Comment:
I added 	s.EqualTimes(currentTime, ai.ScheduledTime) and EqualTimesWithPrecision to persistenceTestBase, So this could be written as:

`s.EqualTimes(expiryTime, ti.ExpiryTime)`